src/components/BinaryButton.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <div class="binary-button">
+    <div @click="onClick(true)" :class="{'binary-button-selected': leftChosen}">
+      <img v-if="leftChosen" class="confirmation-icon" src="../assets/images/check.svg" />
+      <div>
+        {{ choices[0] }}
+      </div>
+      <img v-if="leftChosen" class="confirmation-icon hidden" src="../assets/images/check.svg" />
+    </div>
+    <div @click="onClick(false)" :class="{'binary-button-selected': !leftChosen}">
+      <img v-if="!leftChosen" class="confirmation-icon" src="../assets/images/check.svg" />
+      <div>
+        {{ choices[1] }}
+      </div>
+      <img v-if="!leftChosen" class="confirmation-icon hidden" src="../assets/images/check.svg" />
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    choices: Array
+  },
+  data() {
+    return {
+      leftChosen: true
+    };
+  },
+  methods: {
+    onClick(left) {
+      this.leftChosen = left;
+    }
+  }
+};
+</script>
+
+<style scoped lang="scss">
+  @import "@/assets/scss/_variables.scss";
+  .binary-button {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    flex-grow: 1;
+    width: 438px;
+    height: 36px;
+    border: 1px solid #2BBCC6;
```

Comment:
Should `#2BBCC6` be another color? @einc

Reply:
ah sorry about that! we updated the colors after his PR haha, it should be emGreen!